### ISSUE TITLE
Rename `comments`/`tags`?

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/camtrap-dp#295]
The following terms are currently shared between tables:

- `tags` (deployments, observations)
- `comments` (deployments, media, observations)

When joining tables, they might lead to name clashes or would be assigned a suffix, like `comments.x`. In the R package camtraptor, we will likely give them names such as `deploymentComments`, `observationComments` which - in contrast with other column names - would then differ from their term names in Camtrap DP.

Is it better to make sure that all term names unique across tables (except those that are shared between `event-observations` and `media-observations` as they are in fact the same concepts)? Or should this be handled on the implementation side, potentially leading to different approaches?

Ping @tucotuco @kbubnicki and others

**Comment by tucotuco** [comment]
Having the term names distinct makes them immediately recognizable in any context and implementation, and so I would favor that.

**Comment by peterdesmet** [comment]
In that case I suggest to rename to:

- [ ] deploymentTags
- [ ] deploymentComments
- [ ] mediaComments
- [ ] observationTags
- [ ] observationComments

**Comment by peterdesmet** [comment]
I've suggested the change in #296.

Requiring unique names across tables has the downside that adding new terms in the future might affect existing names. I looked at all term names and I think that in practice, the effect will be limited. Here are the terms names that are the most vulnerable:

- `start`
- `end`
- `timestamp`
- `favorite`
- `count`

I don't see the need to change them now however.
